packages/app/src/hooks/useChartConfig.tsx

```diff
@@ -13,6 +15,49 @@ import { IS_MTVIEWS_ENABLED } from '@/config';
 import { buildMTViewSelectQuery } from '@/hdxMTViews';
 import { getMetadata } from '@/metadata';
 
+export const isMetric = (config: ChartConfigWithOptDateRange) =>
+  config.metricTables != null;
+
+// TODO: apply this to all chart configs
+export const setChartSelectsAlias = (config: ChartConfigWithOptDateRange) => {
+  if (Array.isArray(config.select)) {
+    if (isMetric(config)) {
+      return {
+        ...config,
+        select: config.select.map(s => ({
+          ...s,
+          alias: `${s.aggFn}(${s.metricName})`,
+        })),
+      };
+    }
+    return {
```

Comment:
why are we adding this to non-metrics queries? it's currently breaking the row side panel and it's not clear to me why we need this

Reply:
Ah good catch. I updated that without knowing its impact on other non-charting components (mainly for the consistency of labels between event and metric charts)